### PR TITLE
Edit branch

### PR DESCRIPTION
**Pull request by hugh-burgess**

### CODE REVIEW

**Comment by PhilippAubert** [comment]
LGTM! Can't wait to see it !

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,69 @@
...
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.Navigation {
+  background-color: red;
+  width: 100vw;
+height: 80px;
```
Comment (FriederikeM) [inline src/index.css]: Is your prettier running? Is saw that in a few places in your code the alignment of text is off

File: src/components/Characters.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  }, []);
+  function renderCharacters() {
+  return characters.filter((person) => person.id <= list).map((person) => {
+  const {id, name, image} = person;
+
+ return (
+
+
+        <section key={name} class="characterCard">
```
Comment (FriederikeM) [inline src/components/Characters.js]: not too sure, but it looks like you receive an id from your API (line 27) so i'd set the key to {id} to make it more specific.

**Review by FriederikeM — APPROVED** [review]
Looks good, only left 2 little notes

**Pull request merged**